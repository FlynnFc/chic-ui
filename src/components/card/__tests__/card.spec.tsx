import Card from '..';
import { render } from '../../../test/utils';

/* eslint-disable */
describe('Card Component', () => {
  it('Renders Correctly', () => {
    const CardGroup = render(
      <>
        <Card
          type="secondary"
          title="Title"
          subTitle="Subtitle"
          description="Lorem ipsum dolor sit amet, consectetur"
          footer="Footer"
          href="https://github.com/karishmashuklaa/chic-ui"
          hrefText="Visit site"
        />
      </>
    );
    expect(CardGroup).toMatchSnapshot();
  });
});
