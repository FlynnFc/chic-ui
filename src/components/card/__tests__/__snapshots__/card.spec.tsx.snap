// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component Renders Correctly 1`] = `
<div
  className="sc-bdfBQB giNlPf"
>
  <div
    className="sc-gsTEea eLwBQf"
    type="secondary"
  >
    <h2
      className="sc-hKgJUU qDFE"
    >
      Title
    </h2>
    <h5
      className="sc-eCstlR jXvDce"
    >
      Subtitle
    </h5>
    <p
      className="sc-jSgvzq bhimHd"
    >
      Lorem ipsum dolor sit amet, consectetur
    </p>
    <div
      className="sc-gKseQn cpVRFe"
    >
      <p
        className="sc-iBPTik jbzIgY"
      >
        Footer
      </p>
      <a
        className="sc-fubCzh kbCvRg"
        href="https://github.com/karishmashuklaa/chic-ui"
      >
        Visit site
      </a>
    </div>
  </div>
</div>
`;
